src/Apps/Components/Layouts/LayoutDefault.tsx

```diff
@@ -23,7 +23,7 @@ export const LayoutDefault: FC<BaseLayoutProps> = ({ children }) => {
       >
         <LayoutNav />
 
-        <AppContainer as="main" id="main" flex={1}>
+        <AppContainer as="main" id="main" flex={1} minHeight="100vh">
```

Comment:
Fixes that very annoying issue where on page transition between uncached (client-side) pages the footer sometimes flashes. We never have pages that are less than 100% height, so this just masks the transition for a brief millisecond.